app/grants/templates/grants/hall_of_fame.html

```diff
@@ -0,0 +1,185 @@
+{% load i18n static bundle %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {% include 'shared/head.html' with slim=1 %}
+  {% include 'shared/cards.html' %}
+
+  {% bundle css file hall_of_fame %}
+  <link rel="stylesheet" type="text/x-scss" href={% static "v2/scss/grants/hall_of_fame.scss" %} />
+  {% endbundle %}
+
+  <style>
+    #hof-banner {
+      background-image: url("{% static "v2/images/grants/hall_of_fame/hall_of_fame.svg" %}");
+      background-size: auto 100%;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  </style>
+</head>
+
+<body class="interior {{active}} hall_of_fame g-font-muli">
+  <div class="content__main min-vh-100 d-md-flex flex-column">
+    {% include 'shared/tag_manager_2.html' %}
+    {% include 'shared/top_nav.html' with class='d-md-flex' %}
+    {% include 'grants/nav.html' %}
+
+    <div id="hall-of-fame" class="container-fluid px-0">
+      <div id="hof-banner" class="container-fluid">
+        <div id="hof-banner" class="container content text-left pt-5 pb-5">
+
+          <h2 class="text-center pt-sm-4">{%trans "Grants Hall of Fame" %}</h2>
+
+          <div class="d-block d-sm-none row pt-3 text-violet-400">
+            <div class="col-xs-12 col-sm-6">
+              <h3 class="text-center">
+                {{total_donations}}
+              </h3>
+            </div>
+            <div class="col-xs-12 col-sm-6">
+              <div class="d-none d-sm-block"></div>
+              <h4 class="text-center pt-2">
+                {% trans 'in Total Donations' %}
+              </h4>
+            </div>
+          </div>
+          <div class="d-none d-sm-block row pt-2 text-violet-400">
+            <div class="col-12">
+              <h3 class="text-center">
+                {{total_donations}} {% trans 'in Total Donations' %}
+              </h3>
+            </div>
+          </div>
+          <div class="row pt-3 pt-sm-5">
+            <div class="col-xs-12 col-sm-8 offset-sm-2 text-center text-grey-400 text font-header">
+              Top Gitcoin Grant funders and grantees making impact in the web3 ecosystem.
+            </div>
+          </div>
+          <div class="text-center pt-4 pt-sm-3">
+            <a class="btn btn-primary btn-lg" href="{%url 'grants:new_matching_partner' %}">
+              Join the Funder's League
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="container-fluid px-0">
+        <div id="partner_section" class="container content text-left">
+          <h2 class="text-grey-500 pt-4 pt-sm-5">{%trans "Top Matching Partners" %}</h2>
+          <div class="pt-3"></div>
+
+          <picture>
+            <source media="(max-width: 320px)" srcset="{{ top_matching_partners_mobile_url }}">
+            <source media="(min-width: 321px)" srcset="{{ top_matching_partners_url }}">
+            <img class="w-100" src="{{ top_matching_partners_url }}" alt="Top matching partners">
+          </picture>
```

Comment:
I don't see a need for art direction responsive image here as the are avatars with alway the same image so not getting any vantage on load 2 image with very similar size, scaling the image will fit a lot better, will make less work the admin and the team to upload and will impact better on the performance size.

Reply:
This is not for the avatars.
The design contains 2 sections with images ("Top Matching Partners" and "Top Individual Grant Donors"), and each require a mobile version of the image, according the design.

Reply:
so you plan to handle a banner instead a dynamic are with the partners logos?

Reply:
Yes, that was the plan

Reply:
> @nutrina can we keep the PR template? you will find there some fields that help us review the PR and also a way to us to check what was the issue asking.
> In that way also designers can keep their review here and we can understand if can be merged or not.

I have updated the PR, so that it uses the template.

Reply:
perfect, yes you are right they asked exactly that.

Reply:
looking deep on the design requirement I will suggest you to make questions / propose solutions based on your personal experience, for example I see one comment on the design asking to be png and animated svg upload. How do you see that happening?

Reply:
Yes, I missed the part with "animated svg".
To enable this I will:
1. need to allow the upload of SVGs as images
2. when rendering, depending on the image type (depending on file name extension for example), I will render either an <img> for the png or an <object> for the SVG.

Also I would drop the current approach using art direction, and render the mobile / desktop version of the image using the display helper classes from Bootstrap.

Reply:
What do you think? Does this sound like a good solution?

Reply:
the problem I see is how the internal tem animate svgs, basically using js that makes (for security reasons) not good to upload a svg with js inside.

Reply:
I understand what you mean. In this case the only safe way I would say is to strip the JS form the SVG. But the SVGs would not be animated any more, for this we need that `__SVGATOR_PLAYER__.build` call.

I checked svgator export options. I see that there is also an option to use CSS animations. This should not pose any security threat as far as I know. This however has some limitations: https://www.svgator.com/help/technical-issues/browser-support 

@zacheryschiller I would propose to remove the option to allow uploading SVGs for now (for security reasons), and release this features with images only. And if the animated SVG version is really required, we work on this next, but this will probably mean moving to more restricted animations (using CSS only). How does this sound to you?

Reply:
I did some more research regarding this topic. Both CSS and JS can be potentially malicious, so allowing SVG uploads in this way is not an option.
Is it acceptable to take the more "dry" approach for rendering those SVGs?
What I mean is:
1. Allow the user (admin) to upload to copy and paste the SVG code (which we would sanitise)
2. the admin should also specify the JSON configuration __SVGATOR__.build( ... )
3. We render the sanitised SVG into the HTML, and call the svgator SDK

@octavioamu What do you think of this proposal?
@zacheryschiller Is this acceptable from a users perspective (the user being the admin who uploads this)?

Reply:
hey @nutrina so looks like since this upload will be done by admin (just one person) I'd say let's stick with complete svg for now and then we build on top of that.
This is intended only as a marketing page so let's keep it simple for now and then if needed -> we can look at the option you proposed. 

Would that work ?

Reply:
Sure, this will work.